tests/lib/websocket/test_interest_type.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
+
+from apollo.lib.websocket.interest_type import (InterestTypeFunctionHandler,
+                                                WebSocketObserverInterestType)
+from apollo.models.agent import Agent
+
+
+def test_interest_type_function_handler_run_corresponding_function(db_session):
```

Comment:
```suggestion
def test_interest_type_function_handler_run_agent_listing_function(db_session):
```

or something.

To indicate that this test is very specific to this one interest type. It won't work with others. Each corresponding function will need to be individually tested.